improver/psychrometric_calculations/psychrometric_calculations.py

```diff
@@ -972,8 +901,7 @@ def process(self, cubes):
         data appropriately.
 
         Args:
-        cubes (iris.cube.CubeList or list or iris.cube.Cube):
-            containing:
+            cubes (iris.cube.CubeList or list or iris.cube.Cube) containing:
```

Comment:
I think it is actually going to be a list of cubes (rather than a cubelist) most of the time (e.g. from the CLI bolting several cubes together). It could be a cubelist if someone imported the plugin and used it that way. So I think the docstring needs an "or" to become an "of" as it is in the CLI.

Reply:
Ah that makes more sense.